src/fix.ts

```diff
@@ -24,12 +27,12 @@ export const exportFixer = (
   typedExports: string[],
   regularExports: string[],
   fixer: RuleFixer,
-) => {
+): RuleFix | undefined => {
   try {
-    const source =
-      node.source && (node.source as any).raw
-        ? 'from ' + (node.source as any).raw
-        : '';
+    let source = '';
+    if (node.source !== null && 'raw' in node.source) {
+      source = `from ${node.source.raw}`;
+    }
```

Comment:
Default source to blank and only check if 'raw' is in source if source is not null (prevents type linting errors).